docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+# Becoming an Attestation Provider
+
+This guide is a walk through to become an [attestation provider](#attestation-provider).
+There are two situations in which it makes sense to do so:
+
+* Running local attestation providers, without any capital requirement, to act as a safeguard against malicious forks ;
+* Running an incentivized attestation provider to reap rewards, which contributes to the network and helps improve the decentralization of the network and provides attestations for the [state connector](#state-connector).
+
+??? note "Rewards"
+
+    Currently, the process of rewarding public attestation providers is still being discussed.
+
+## Prerequisites
+
+Before proceeding with the [installation](#installation), it is essential to follow every prerequisite.
+
+### System Requirements
+
+The following specifications were observed to be able to run all nodes on a single machine.
+Ideally, a single node should be run on each system in production for better performance and uptime.
+
+* **OS**: Ubuntu 20.04
+* **CPU Cores**: 16
+* **RAM**: 96GB
+* **Disk space**: 3TB with an option or plan in place to expand capacity. SSD is recommended because some chains like xrpl can fall out of sync on regular disks.
+
+As of Q2 2022, this is roughly what you can expect from each node regarding disk usage:
+
+| Volume              | Size  |
+|---------------------|-------|
+| algorand-data       | 1TB   |
+| algorand-indexer-db | 1TB   |
+| bitcoin-data        | 500GB |
+| dogecoin-data       | 60GB  |
+| litecoin-data       | 90GB  |
+| ripple-data         | 230GB |
+
+We are looking into ways to run a non-full history algorand node.
+
+### Dependencies
+
+* [`git`](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [`docker`](https://docs.docker.com/get-docker/)
+* [`docker-compose`](https://docs.docker.com/compose/install/)
+* [`openssl`](https://www.openssl.org/source/)
+* [`python3`](https://www.python.org/downloads/)
+
+### Node Setup
+
+The first step is to set up nodes for each supported network, that the attestation provider suite can then use to make attestations.
```

Comment:
```suggestion
The first step is to set up a node for each supported network, which the attestation provider suite can then use to make attestations.
```

Reply:
Fixed in [`fd8533f` (#14)](https://github.com/flare-foundation/docs/pull/14/commits/fd8533fd665a5d20581c49e615679893b47cd2b5)